testing/megatools/APKBUILD

```diff
@@ -8,12 +8,17 @@ url="https://megatools.megous.com"
 arch="all"
 license="GPL2"
 depends="libssl1.0"
```

Comment:
libssl runtime dependency should be autodetected by abuild so `depends` should be empty